### PR TITLE
docs: added steps to setup reCAPTCHA

### LINKED ISSUE #53: docs: add steps to setup recaptcha (optional local dev environment setup)
### Description

Developers need to setup thier ReCaptcha local environment variable so that they dont run into these errors in `/register` and `/faq` page.

image

-----------------------------

update documentation( contributing.md) --- dev environment setup part. mark it as "optional setup" or maybe "additional setup"(call whatever you like)

Write the steps to setup and create thier own ReCaptcha local environment variables. (Its all about going to google's recaptcha site and doing some stuff there and creating a .local.env file locally to setup ReCaptha feature)

refer https://prateeksurana.me/blog/integrating-recaptcha-with-next/ for the steps.

### PR DESCRIPTION
**Pull request by akshaaatt**
- Fixes #53

### CODE REVIEW

File: .docs/CONTRIBUTING.md [hunk 1]
```diff
@@ -90,6 +90,28 @@ The following tools should be installed and running on your computer:
...
+Our app uses reCAPTCHA to help us prevent spammers/bots for polluting our database and helps us enhance the security we provide on our app.
+In order to access the `/register` and `/faq` pages on the app seamlessly, follow the required steps:
+
+1. Head to the [reCAPTCHA admin console](https://www.google.com/recaptcha/admin/create) and fill in the required details.
+
+![recaptcha-registration-1200](https://user-images.githubusercontent.com/51470769/193463737-5e6eda09-2015-4fec-93a0-a763a7b0993e.jpg)
+
+2. After clicking submit, you should be able to see the public and secret keys.
+
+![recaptcha-keys-1200](https://user-images.githubusercontent.com/51470769/193463777-b2438d33-3053-4811-9be2-9c6eae000d2a.jpg)
```
Comment (ashuvssut) [inline .docs/CONTRIBUTING.md:104]: there is a assets folder in .docs folder

put these images in that assets folder only

**Review by ashuvssut — CHANGES_REQUESTED** [review]

**Comment by ashuvssut** [comment]
Make those changes, rest looks perfect! Thanks

**Comment by ashuvssut** [comment]
Also I was wondering about you commit message...

I mean its not an issue, you dont need to change it. I just that I have setup git-hooks that prevent those kind of commit messages.

Those git hooks will not let you commit unless you follow the proper way to write the commit messages( as mentioned in GIT-GUIDELINES.md). It will show an error instead to help you write the correct message

Did you encounter those message or did you use `--no-verify` flag to bypass the git hooks? Do you think those hooks did not work?

File: .docs/CONTRIBUTING.md (hunk 1 above)
Comment (akshaaatt) [reply .docs/CONTRIBUTING.md]: Fixed

**Comment by akshaaatt** [comment]
@ashuvssut I faced the commit lint right now when I had to commit locally. Previously, I had used the GitHub interface itself to add the documentation. Fixed the details now to make the recent commit.

**Review by ashuvssut — APPROVED** [review]

**Pull request merged**